tests/testthat/TestS4/test.Rproj

```diff
@@ -5,12 +5,13 @@ SaveWorkspace: No
 AlwaysSaveHistory: Default
```

Comment:
This file should be removed from git (not your fault though I am sure I inadvertently added it).